### PR TITLE
80606 Email changes

### PR DESCRIPTION
**Pull request by bratli**
DevOps#80606

Reformat how to display info in email

### CODE REVIEW

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        public static string GenerateMeetingTitle(Invitation invitation) 
+            => $"Invitation to Punch Out, IPO-{invitation.Id}, Project: {invitation.ProjectName}";
+
+        public static string GenerateMeetingDescription(Invitation invitation, string baseUrl)
+        {
+            var meetingDescription = "<h4>You have been invited to attend a punch round.</h4>";
+            meetingDescription += $"<p>Title: {invitation.Title}</p>";
+            meetingDescription += $"<p>Type: {invitation.Type}</p>";
+            var location = string.IsNullOrEmpty(invitation.Location) ? "-" : invitation.Location;
```
Comment (hbjorgo) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs:15]: Consider using string.IsNullOrWhitespace(...) here and 2 lines below.

**Review by hbjorgo — APPROVED** [review]

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs [hunk 2]
```diff
@@ -439,39 +439,19 @@ private async Task AddMcPkgsAsync(Invitation invitation, IList<string> mcPkgScop
         {
             var meeting = await _meetingClient.CreateMeetingAsync(meetingBuilder =>
             {
+                var baseUrl = _meetingOptions.CurrentValue.PcsBaseUrl +
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs:442]: Use string interpolation + ensure this to work if config misses the / at end of csBaseUrl:
var baseUrl = $"{_meetingOptions.CurrentValue.PcsBaseUrl.Trim('/')}/{_plantProvider.Plant.Substring(4, _plantProvider.Plant.Length - 4).ToUpper()}";

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs [hunk 3]
```diff
@@ -74,14 +74,16 @@ public async Task<Result<string>> Handle(EditInvitationCommand request, Cancella
             invitation.SetRowVersion(request.RowVersion);
             try
             {
+                var baseUrl = _meetingOptions.CurrentValue.PcsBaseUrl +
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs:77]: as prev comment

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands
+{
+    public class MeetingInvitationHelper
+    {
+        public static string GenerateMeetingTitle(Invitation invitation) 
+            => $"Invitation to Punch Out, IPO-{invitation.Id}, Project: {invitation.ProjectName}";
+
+        public static string GenerateMeetingDescription(Invitation invitation, string baseUrl)
+        {
+            var meetingDescription = "<h4>You have been invited to attend a punch round.</h4>";
```
Comment (janid1967) [inline src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs:12]: Consider use of @ to make a string of the whole email template (which can be moved to a sep file later). Add "tokens" inside the string-template, such as [InvitationTitle], and run a .Replace of known tokens: .Replace("[InvitationTitle]", invitation.Title)

This goes for all the string making html here. I can assist to show what I mean

**Review by janid1967 — APPROVED** [review]
Approved with suggestions

File: src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs (hunk 4 above)
Comment (janid1967) [reply src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs:12]: image

Comment (janid1967) [reply src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs:12]: var meetingDescription = 
                @"
                        <h4>You have been invited to attend a punch round.</h4>
                        <p>Title: [InvitationTitle]</p>
                        <p>Type: [InvitationType]</p>
                        <p>Location: [Location]</p>
                        <p>Description: [Description]</p>
                        <p>Scope: </p>
                        [McPkgTable]
                        [CommPkgTable]
                        </br><a href='[Url]'>Open invitation for punch out in ProCoSys.</a>
                ";

Comment (bratli) [reply src/Equinor.ProCoSys.IPO.Command/InvitationCommands/MeetingInvitationHelper.cs:12]: Okay, yes lets talk about how to do it

**Pull request merged**